app/views/hotels/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h2> Hotel Info </h2>
+<% @hotel.rooms.each do |room| %>
+  <%# <% binding.pry %> 
+  <p>Unique guests: <% room.guests.each do |guest| %></p>
+    <%= guest.name %>
+  <% end %>
+<% end %>
```

Comment:
You would have wanted to create a `::unique_guests` method on the `Hotel` model and loop through that.